src/main/java/com/minecolonies/coremod/colony/Colony.java

```diff
@@ -335,11 +333,13 @@ private void readFromNBT(@NotNull final NBTTagCompound compound)
         {
             final NBTTagCompound achievementCompound = achievementTagList.getCompoundTagAt(i);
             final String achievementKey = achievementCompound.getString(TAG_ACHIEVEMENT);
-            final StatBase statBase = StatList.getOneShotStat(achievementKey);
-            if (statBase instanceof Achievement)
+
+            //todo serialization
```

Comment:
![INFO](https://sonarsource.github.io/sonar-github/severity-info.png 'Severity: INFO') Complete the task associated to this TODO comment. [![rule](https://sonarsource.github.io/sonar-github/rule.png)](https://sonar.minecolonies.com/coding_rules#rule_key=squid%3AS1135)